src/installer/__main__.py

```diff
@@ -84,6 +92,8 @@ def _main(cli_args: Sequence[str], program: Optional[str] = None) -> None:
         bytecode_levels = [0, 1]
 
     with WheelFile.open(args.wheel) as source:
+        if args.validate_record != "none":
+            source.validate_record(args.validate_record == "all")
```

Comment:
Did you forget to handle the `"entries"` case here? Right now, it seems the current behavior of `--validate-record=entries` is the same as `--validate-record=none`.

Reply:
When passing `entries` instead of `none`, basic record format validations will be performed (as of #156), instead computing and comparing file hashes (which may cause perf issues when the files are large).

Reply:
`WheelFile.validate_record()`'s default is validating hashes though.